.github/workflows/ci.yml

```diff
@@ -37,20 +38,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-py${{ matrix.python }}-
       - name: Install dev dependencies
-        run: pip install --requirement requirements-dev.txt
+        run: >
+          pip install
+          --requirement requirements-dev.txt
 
       # Install library
       - name: Install result
         run: pip install --editable .
 
       # Tests
       - name: Run tests
-        run: pytest --cov-report term --cov-report xml:coverage.xml --ignore-glob=result/tests-pattern-matching.py
+        run: >
+          pytest
+          --ignore=tests/test_pattern_matching.py
       - name: Run tests (pattern matching)
-        run: pytest --cov-report term --cov-report xml:coverage.xml result/tests-pattern-matching.py
```

Comment:
Is the removal of `--cov-report` okay? Coverage reporting still works as expected?

Reply:
all flags have moved into `pyproject.toml` as defaults which makes them work locally as well. here's my editor showing that the overloads are not executed 😄 

![image](https://user-images.githubusercontent.com/748944/139555104-072e8d8a-1194-446b-b134-fa05c42c4e0e.png)